knowledge_repo/app/templates/auth-login-form.html

```diff
@@ -6,7 +6,8 @@
 .login-form {
     display: block;
     list-style-type: none;
-    margin: 10px;
+    width: 340px;
```

Comment:
Can we separate the cosmetic changes from the LDAP patch? Or if its a clear improvement, can you provide a screenshot of the improvement?

Reply:
Here's a screenshot of the cosmetic changes. I think it's a clear improvement, below you can see a comparison between the two:

New:
![screenshot-2018-5-23 knowledge repo login](https://user-images.githubusercontent.com/7283679/40408435-cab37a48-5e67-11e8-89a5-fe16f3919b44.png)

Old:
![screenshot-2018-5-23 knowledge repo login 1](https://user-images.githubusercontent.com/7283679/40408493-00ed0480-5e68-11e8-8054-4ba112f96f25.png)

Reply:
This is indeed an improvement! Leave it in as part of this patch :).